Chaincase.UI/Pages/SelectCoinsToJoinPage.razor

```diff
@@ -0,0 +1,106 @@
+@page "/coinjoin/selectcoins"
+@page "/coinjoin/selectcoins/{ReqInputAmountBTC}"
+@using NBitcoin
+@using System.Reactive.Linq
+@using Chaincase.Common
+@using Chaincase.UI.Services
+@inject UIStateService UiStateService
+@inject NavigationManager NavigationManager
+@inject StackService StackService
+@inject SelectCoinsViewModel SelectCoinsViewModel
+@inject CoinJoinViewModel CoinJoinViewModel 
+@inherits ReactiveComponentBase<SelectCoinsViewModel>
+
+
+<IonContent>
+    @if (!ViewModel.CoinList.Any())
+    {
+        <IonText class="ion-text-center">
+            <p>Your coins will appear here</p>
+        </IonText>
+    }
+    else
+    {
+        <IonList>
+            @foreach (var coin in ViewModel.CoinList)
+            {
+                <IonItemSliding>
+                    <IonItem button>
+                        <IonCheckbox slot="start"
+                                     @bind-Value="coin.IsSelected" />
+                        <IonLabel slot="start">
+                            <h3>₿ @coin.Amount</h3>
+                            <p>@coin.Clusters</p>
+                        </IonLabel>
+                        <IonNote slot="end" color="@(coin.AnonymitySet > Config.DefaultPrivacyLevelSome ? "success" : "warning")">
+                            <ion-anchor @onclick="@(() => OpenCoinDetail(coin))">@coin.AnonymitySet</ion-anchor>
+                        </IonNote>
+                    </IonItem>
+                    <IonItemOptions side="end">
+                        <IonItemOption OnClick="@(() => OpenCoinDetail(coin))" color="default">Details</IonItemOption>
+                    </IonItemOptions>
+                </IonItemSliding>
+            }
+        </IonList>
+    }
+
+</IonContent>
+<IonFooter>
+    <IonCard color="light">
+        <IonCardContent>
+            @if (ReqInputAmount() is not null)
+            {
+                <IonText color="medium">
+                    <p>
+                        <strong>Req. Inputs</strong>
+                        <span class="ion-float-end">@ReqInputAmount() BTC</span>
+                    </p>
+                </IonText>
+
+            }
+            <IonText color="primary">
+                <p>
+                    <strong>Total Inputs</strong>
+                    <span class="ion-float-end">@ViewModel.SelectedAmount BTC</span>
+                </p>
+            </IonText>
+            <IonButton class="neu-button" OnClick='() => IsAuthorizeModalVisible = true' color="primary" expand="block" disabled="@(!HasSelectedEnough)">CONTINUE</IonButton>
+        </IonCardContent>
+    </IonCard>
+</IonFooter>
```

Comment:
Perhaps the Content & Footer could be re-usable components for both select here & during a send